src/components/breadcrumbs/template.test.js

```diff
@@ -0,0 +1,112 @@
+/* globals describe, it, expect */
+
+const { render, getExamples } = require('../../../lib/jest-helpers')
+
+const examples = getExamples('breadcrumbs')
+
+describe('Breadcrumbs', () => {
+  describe('by default', () => {
+    it('renders with classes', () => {
+      const { $ } = render('breadcrumbs', {
+        classes: 'app-c-breadcrumbs--custom-modifier'
+      })
+
+      const $component = $('.govuk-c-breadcrumbs')
+      expect($component.hasClass('app-c-breadcrumbs--custom-modifier')).toBeTruthy()
+    })
+
+    it('renders with attributes', () => {
+      const { $ } = render('breadcrumbs', {
+        attributes: {
+          'id': 'my-navigation',
+          'role': 'navigation'
+        }
+      })
+
+      const $component = $('.govuk-c-breadcrumbs')
+      expect($component.attr('id')).toEqual('my-navigation')
+      expect($component.attr('role')).toEqual('navigation')
+    })
+
+    it('renders with items', () => {
```

Comment:
Looks like items are covered below so we may not need this additional test.

Reply:
It kinda' is, but if `renders item with text` is failing is not explicit why. may have an item without content or you may not have the item at all. that was the logic behind it. happy to remove it if others feel the same about it being redundant.